tc-messaging/src/main/java/com/tc/io/TCByteBufferInputStream.java

```diff
@@ -316,11 +327,7 @@ private void nextBuffer() {
 
   @Override
   public void reset() {
```

Comment:
Can we remove this method (and the associated mark() method) from the interface now?

Reply:
yup